src/components/BoardHeader/BoardHeader.scss

```diff
@@ -63,10 +63,11 @@
 .board-header__share-button {
   color: $color-backlog-blue;
   outline: none;
-  border: 2px solid $color-backlog-blue;
+  border: 2px solid #333948;
+  color: #333948;
   border-radius: 100%;
-  height: $icon--extralarge;
-  width: $icon--extralarge;
+  height: 36px;
+  width: 36px;
```

Comment:
I guess 32px is too small? (in case it's not `$icon--large: 32px;`)

Reply:
I think it's too small. Looks strange in contrast to the larger avatars